src/main/java/com/oltpbenchmark/benchmarks/ycsb/YCSBBenchmark.java

```diff
@@ -43,6 +43,11 @@ public class YCSBBenchmark extends BenchmarkModule {
      */
     protected final int fieldSize;
 
+    /**
+     * The constant used in the zipfian distribution (to modify the skew)
+     */
+    protected final double zipfianConstant;
```

Comment:
Can you rename the class member to `skewFactor` too?

Reply:
@apavlo Done.